gui/mozregui/build_runner.py

```diff
@@ -74,7 +74,9 @@ def evaluate(self, build_info, allow_back=False):
         else:
             self.evaluate_started.emit('')
 
-    def finish(self, verdict):
+    def finish(self, verdict, options):
+        if options['profile'] and options['profile_persistence'] == 'clone-first':
+            options['profile'].cleanup()
```

Comment:
Yes, I would change that. Yes, it will work. It will evaluate to `None` in that case.